packages/dialog/src/uxp-dialog-base.css

```diff
@@ -11,3 +11,7 @@ governing permissions and limitations under the License.
 */
 
 /* write uxp style overrides */
+
+:host {
+    height: 100vh;
```

Comment:
Can you please add one line comment about each css override in the PR description?